appveyor.yml

```diff
@@ -1,14 +1,12 @@
-init:
-  - git config --global core.autocrlf true
-branches:
-  only:
-    - master
-    - release
-    - dev
-    - /^(.*\/)?ci-.*$/
+image: Windows Server 2016
+
+install:
+  - ps: Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.11.2/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
```

Comment:
I thought the same, but later I realized it is already installed and listed in https://www.appveyor.com/docs/installed-software/#tools